dev-requirements.txt

```diff
@@ -32,10 +32,13 @@ more-itertools>=4.0.0; python_version >= '3.6'
 # safety 1.9.0 removed support for Python 2.7 (and now also enforces that)
 safety>=1.8.7,<1.9.0; python_version == '2.7'
 safety>=1.9.0; python_version >= '3.5'
+safety>=2.2.0; python_version >= '3.6'
 # dparse 0.5.0 has an infinite recursion issue on Python 2.7,
 #   see https://github.com/pyupio/dparse/issues/46
 dparse>=0.4.1,<0.5.0; python_version == '2.7'
 dparse>=0.5.1; python_version >= '3.5'
+# ver 0.6.2 min requirement by safety 2.2.0
+dparse>=0.6.2; python_version >= '3.6'
```

Comment:
If you add this for py>=3.6, you need also to limit the previous line to py<3.6, i.e. py==3.5.

Reply:
DONE